src/index.js

```diff
@@ -40,6 +40,15 @@ const _init = Symbol('init');
 const _defineSetting = Symbol('_defineSetting');
 const _internalSet = Symbol('_internalSet');
 
+// In order to prevent an unrestricted number of event listeners on Process, we create the listeners here
+// to close the database on exit.
+const _instances = [];
+process.on('exit', () => {
+  for (let instance of _instances) {
```

Comment:
change let to const as it isn't redefined

Reply:
just a habit sorry!